mytimer/functions.py

```diff
@@ -123,13 +128,21 @@ def inner_function(
         :type tone: int
         :param alarm_repeat: alarm repeat
         :type alarm_repeat: int
+        :param v_shift: vertical shift
+        :type v_shift: int
+        :param h_shift: horizontal shift
+        :type h_shift: int
         :return: None
         """
         message = message.strip()
         if len(message) > 0:
             message = MESSAGE_TEMPLATE.format(message)
         if alarm_repeat < 1:
             alarm_repeat = 1
+        if h_shift < 0 or not isinstance(h_shift, int):
```

Comment:
We can remove the use of `isinstance` since we are already checking the type of this parameter in the CLI parser.